Description: Link flang-driver (clang) against Debian's LLVMPolly.so
 Needs to be applied after driver tarball is unpacked
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2018-11-29
Forwarded: not-needed

Index: flang-flang_20180921/flang-driver/tools/driver/CMakeLists.txt
===================================================================
--- flang-flang_20180921.orig/flang-driver/tools/driver/CMakeLists.txt
+++ flang-flang_20180921/flang-driver/tools/driver/CMakeLists.txt
@@ -123,5 +123,5 @@ if(CLANG_ORDER_FILE AND
 endif()
 
 if(WITH_POLLY AND LINK_POLLY_INTO_TOOLS)
-  target_link_libraries(clang PRIVATE Polly)
+  target_link_libraries(clang PRIVATE /usr/lib/llvm-7/lib/LLVMPolly.so)
 endif(WITH_POLLY AND LINK_POLLY_INTO_TOOLS)
